File: week1/tripCost.py
"""
    Description: This program will compute the cost of a multi-day trip for one
                 or more people.
    Author: Jonas Pfefferman '24
    Date: 9/3/20
"""

def main ():

    print("Let's go on a trip!\n")

    # Getting trip info: travel cost, food/lodging cost, days, number of people
    personTravelCost = float(input("What is the travel cost per person? "))
    personFoodCost = float(input("What is the food/lodging cost per person per day? "))
    travelDays = int(input("How many days will you be traveling? "))
    numberPeople = int(input("How many people are going? "))

    # Calculations for single person
    personTotalFoodCost = float(personFoodCost * travelDays)
    personTotalBill = float(personTotalFoodCost + personTravelCost)

    # calculations for total
    totalTravelCost = float(personTravelCost * numberPeople)
    peopleFoodCost = float(personFoodCost * numberPeople)
    totalFoodCost = float(peopleFoodCost * travelDays)
    totalBill = float(totalTravelCost + totalFoodCost)

    # Turning Floats/Integers into Strings
    personTotalBillString = str(personTotalBill)
    totalBillString = str(totalBill)


    # Printing results
    print("Cost per person: ", "$" + personTotalBillString)
    print("Total cost: ", "$" + totalBillString)



main()

File: AP/week1/tripCost.java
/*
Description: This program will computer the cost of a multi-day trip for one or
             more people.
Author: Jonas Pfefferman '24
Date: 1/29/21
*/

import java.util.Scanner;

public class tripCost {
    public static void main(String[] args) {
        Double personTravel;
        Double personFood;
        Integer travelDays;
        Integer numberPeople;
        Double personTotalBill;
        Double totalBill;
        Scanner personTravelCost;
        Scanner personFoodCost;
        Scanner days;
        Scanner numberOfPeople;

        personTravelCost = new Scanner(System.in);
        System.out.println("What is the travel cost per person?");
        personTravel = personTravelCost.nextDouble();

        personFoodCost = new Scanner(System.in);
        System.out.println("What is the food/lodging cost per person per day?");
        personFood = personFoodCost.nextDouble();

        days = new Scanner(System.in);
        System.out.println("How many days will you be traveling?");
        travelDays = days.nextInt();

        numberOfPeople = new Scanner(System.in);
        System.out.println("How many people are going?");
        numberPeople = numberOfPeople.nextInt();

        personTotalBill = (personFood * travelDays) + personTravel;

        totalBill = personTotalBill * numberPeople;

        System.out.println("Cost per person: $" + personTotalBill);
        System.out.println("Total cost: $" + totalBill);
    }
}